frontend/scripts/metrica.js

```diff
@@ -0,0 +1,231 @@
+(() => {
+    let state = {
+        host: 'http://stage.metrica.fun/api/v1',
```

Comment:
I think it should be inside .env file, but leave it like this for a while, until I setup minifier and babel, as you use ES6 we need to compile it for more ancient browsers.

Reply:
This was a temporary solution, as there is no script buildyet, we cannot use the value from .env